core/src/test/scala/org/apache/spark/deploy/rest/StandaloneRestSubmitSuite.scala

```diff
@@ -440,7 +440,6 @@ class StandaloneRestSubmitSuite extends SparkFunSuite with BeforeAndAfterEach {
     val mainJar = "dummy-jar-not-used.jar"
     val commandLineArgs = Array(
       "--deploy-mode", "cluster",
-      "--master", masterUrl,
       "--name", mainClass,
```

Comment:
I think this is a useless setting, it will be rewritten in rest server side with right value.